bats_ai/core/views/recording.py

```diff
@@ -127,12 +127,14 @@ def get_spectrogram_compressed(request: HttpRequest, id: int):
         return {'error': 'Recording not found'}
 
     spectrogram = recording.spectrogram
-    compressed, starts, ends = spectrogram.compressed
+    compressed, starts, ends, duration = spectrogram.compressed
 
     spectro_data = {
         'base64_spectrogram': compressed,
         'spectroInfo': {
             'width': spectrogram.width,
+            'start_time': 0,
+            'end_time': duration,
```

Comment:
this line can be updated to simply use `spectrogram.duration` and removing the fourth return value from the compressed function.  If the total duration of the compressed spectrogram is needed, the return value for duration is incorrect.

Reply:
Thanks, I've changed it to just using `specotrgram.duration`  (That is the number I want).  The start_time is there because in the future we may want to do portions of a clip instead of the full length.